src/interaction/interactiveTarget.js

```diff
@@ -42,5 +49,11 @@ module.exports = {
      * @todo Needs docs.
      * @private
      */
-    _touchDown: false
+    _touchDown: false,
+    
+    /**
+     * Used to track the global start position of a touchdown event on this target.
+     * @private
+     */
+    _touchDownPoint: new core.Point()
```

Comment:
Not sure I understand why the touchdown point is on the object being touched? Seems like this makes more sense to be a property in the manager.